Makefile

```diff
@@ -213,17 +211,38 @@ configure: whoami
 build: $(call make_help,build,Builds the source into an image in the tools project namespace)
 build: OC_PROJECT=$(OC_TOOLS_PROJECT)
 build: whoami
-	$(call oc_build,$(PROJECT_PREFIX)etl)
+	$(call oc_build,$(PROJECT_PREFIX)ggircs-etl)
+
+GGIRCS_DB_NAME = "ggircs"
+GGIRCS_USER_NAME = "ggircs"
+
 
 .PHONY: install
 install: whoami
-	$(call oc_promote,$(PROJECT_PREFIX)etl)
-	$(call oc_wait_for_deploy_ready,$(PROJECT_PREFIX)postgres)
+	$(eval GGIRCS_PASSWORD = $(shell if [ -n "$$($(OC) -n "$(OC_PROJECT)" get secret/cas-ggircs-postgres --ignore-not-found -o name)" ]; then \
+$(OC) -n "$(OC_PROJECT)" get secret/cas-ggircs-postgres -o go-template='{{index .data "database-password"}}' | base64 -d; else \
+openssl rand -base64 32 | tr -d /=+ | cut -c -16; fi))
+	$(eval OC_TEMPLATE_VARS += GGIRCS_PASSWORD="$(shell echo -n "$(GGIRCS_PASSWORD)" | base64)" GGIRCS_USER="$(shell echo -n "ggircs" | base64)" GGIRCS_DB="$(shell echo -n "ggircs" | base64)")
+	$(eval PREVIOUS_DEPLOY_SHA1 = $(shell $(OC) -n "$(OC_PROJECT)" get job $(PROJECT_PREFIX)ggircs-etl-deploy --ignore-not-found -o go-template='{{index .metadata.labels "cas-pipeline/commit.id"}}'))
```

Comment:
we're probably going to want to clean this up eventually...in theory we should be able to put it in named macros that explain what it's doing